Source/JavaScriptCore/dfg/DFGObjectAllocationSinkingPhase.cpp

```diff
@@ -1985,8 +1985,27 @@ class ObjectAllocationSinkingPhase : public Phase {
         // they are inserted only once and we don't clutter the graph
         // with useless constants everywhere
         UncheckedKeyHashMap<FrozenValue*, Node*> lazyMapping;
-        if (!m_bottom)
-            m_bottom = m_insertionSet.insertConstant(0, m_graph.block(0)->at(0)->origin, jsNumber(1927));
+        m_bottom = m_insertionSet.insertConstant(0, m_graph.block(0)->at(0)->origin, jsNumber(1927));
```

Comment:
Why remove the conditional init for `m_bottom`? Are we inserting a new bottom value here every time.

Reply:
This is the first time m_bottom is reached so it was always set here.